src/modules/expression-evaluator/src/evaluateExpression.ts

```diff
@@ -105,15 +105,17 @@ export default async function evaluateExpression(
         }, origString)
 
       case 'API':
-        let url, urlWithQuery, queryFields, queryValues: string[], returnProperty
+        let urlWithQuery, returnedProperty
         try {
-          ;[url, queryFields, queryValues, returnProperty] = assignChildNodesToQuery(
-            childrenResolved
-          )
+          const { url, fieldNames, values, returnProperty } = assignChildNodesToQuery([
+            '', // Extra unused field for API (query)
+            ...childrenResolved,
+          ])
+          returnedProperty = returnProperty
```

Comment:
Just wondering why the need to rename the variable? Why not just keep using `returnProperty` all the way?

Reply:
Not accessible when Ii changed this to be a destructed object (received inside the `try` block), not the prettiest like that I agree.

Reply:
Ah yes, I see. I've come across that particular problem in a few places, actually. Wish there was a more elegant solution for it.